client/signup/steps/design-picker/index.jsx

```diff
@@ -384,7 +388,8 @@ export default function DesignPickerStep( props ) {
 			{ ...props }
 			className={ classnames( {
 				'design-picker__has-categories': showDesignPickerCategories,
-				'design-picker__sell-intent': 'sell' === intent || isDIFMStoreFlow,
+				'design-picker__sell-intent': 'sell' === intent,
+				'design-picker__hide-category-column': useDIFMThemes,
```

Comment:
From the CSS below, these two classes are doing the same thing. Could we unify them as one? e.g. 
```
design-picker__hide-category-column: useDIFMThemes || 'sell' === intent
```

Reply:
Nice catch! Fixed via https://github.com/Automattic/wp-calypso/pull/80419/commits/1873556ad36fa6e03d5fd04ab82883d17013a074

Reply:
LGTM 🚢